CONTRIBUTING.md

```diff
@@ -258,3 +258,19 @@ the main branch. The workflow file should be named as
 [![NuGet version badge](https://img.shields.io/nuget/v/{your_package_name})](https://www.nuget.org/packages/{your_package_name})
 [![NuGet download count badge](https://img.shields.io/nuget/dt/{your_package_name})](https://www.nuget.org/packages/{your_package_name})
 ```
+
+### Guidance for target frameworks
+
+Components shipped from this repo SHOULD comply with the following requirements:
+
+* SHOULD support all the officially supported versions of
```

Comment:
Should we point to the supported versions of main repo instead and say that?

* SHOULD support the [.NET versions](https://github.com/open-telemetry/opentelemetry-dotnet/tree/main#supported-net-versions) which are supported by the OpenTelemetry main repo packages.

Reply:
done